integration_tests/features/TransactionInfo.feature

```diff
@@ -0,0 +1,26 @@
+@merge-mining
+    Feature: Transaction Info
+
+    @critical
+        Scenario: Get Transaction Info
+            Given I have a seed node NODE
+            And I have 1 base nodes connected to all seed nodes
+            And I have wallet WALLET_A connected to all seed nodes
+            And I have wallet WALLET_B connected to all seed nodes
+            And I have a merge mining proxy PROXY connected to NODE and WALLET_A
+            When I merge mine 2 blocks via PROXY
+            Then all nodes are at height 2
+            And I send 1000000 tari from WALLET_A to one wallet WALLET_B at fee 100
+            Then Transaction status of last result from WALLET_A to WALLET_B is known to both wallets
```

Comment:
A better thing to test would be something like 
`Then WALLETA sees the last transaction as MINED`
`And WALLETB sees the last transaction as MINED`

Reply:
This is not a robust approach as it depends on other network conditions, as the only thing we are interested in here is if the transaction status can be retrieved for both wallets, irrespective of what it may be.